pkg/storage/inmemory/backend.go

```diff
@@ -52,11 +54,12 @@ type Backend struct {
 	closeCtx context.Context
 	close    context.CancelFunc
 
-	mu       sync.RWMutex
-	lookup   map[string]storage.RecordCollection
-	capacity map[string]*uint64
-	changes  *btree.BTree
-	leases   map[string]*lease
+	mu            sync.RWMutex
+	lookup        map[string]storage.RecordCollection
+	lookupOptions map[string]*databroker.Options
+	capacity      map[string]*uint64
```

Comment:
do we need `capacity` as a separate map? Or is it easier to just use `lookupOptions` for that too?